GitLab now supports 2FA (software: OTP)

rugk opened the pull request:
GitLab [let me know](https://twitter.com/gitlab/status/604405976884731905) that they are now supporting 2FA, so here I added it.
For now I've added their blog post as a documentation.

rugk committed:
GitLab changed support to yes (OTP)

Carlgo11 commented on _data/developer.yml:
```diff
@@ -101,7 +101,9 @@ websites:
       url: https://gitlab.com
       twitter: gitlab
```
remove the twitter handle please

rugk replied:
Done.

rugk committed:
Removed twitter username

rugk committed:
Added restriction (LDAP)

Source: https://about.gitlab.com/2015/05/22/gitlab-7-11-released/#comment-2039678391
